### DISCUSSION TITLE
Welcome to v2ray-agent Discussions!

### DISCUSSION THREAD

**Discussion by mack-a** [mack-a/v2ray-agent#510] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joostshao** [comment]
😘 , [mack-a](https://github.com/mack-a)

**Comment by WhiteByeBye** [comment]
Wow, finally, you come back.

**Comment by Zxis233** [comment]
GOD YOU FINALLY COME BACK😭

**Comment by alirezarah** [comment]
Hi
cdn doesn't work in some configurations, please upgrade the cdn part, you are really great

**Comment by hyarian** [comment]
Hi
first of all thanks for your script, it runs smooth and fast and without problem for me.
is it possible for you to add a feature to limit connection per uid? (only one or two device be able to connect to server per UID) there is a script [here ](https://github.com/net-pioneer/v2ray-connection-limiter/blob/main/main.py)that works with x-ui , but it call the UID and ports from database and i think your script doesn't create any database. is there anyway to achieve this?
